### PR TITLE
QR Code Generation and Event Page

### PR DESCRIPTION
**Pull request by LightningFrisk**

### CODE REVIEW

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
   <h1> Welcome to ScoreKeeper</h1>
     <a href="user/join">Join a Tournament</a>
     <a href="user/host">Host a Tournament</a>
+<!--    <a href="qrcode">Test QR Codes</a>-->
```
Comment (LightningFrisk) [inline src/main/resources/templates/index.html:12]:
```suggestion

```

File: src/main/java/org/launchcode/scorekeeperapp/controllers/BarcodesController.java
Comment (LightningFrisk) [inline src/main/java/org/launchcode/scorekeeperapp/controllers/BarcodesController.java:1]: This controller is actually unneeded but is a good proof of concept for further use if we need it

File: build.gradle
Comment (LightningFrisk) [inline build.gradle:1]: These are what enable the QR Gen stuff to work.

**Review by jghilliard — APPROVED** [review]
QR Code generation works for me!

**Review by monivg — APPROVED** [review]
QR Code works! Great Job Caleb!

**Pull request merged**